core/src/main/java/greencity/exception/handler/RetrieveMessageErrorDecoder.java

```diff
@@ -0,0 +1,47 @@
+package greencity.exception.handler;
+
+import feign.Response;
+import feign.codec.ErrorDecoder;
+import greencity.exceptions.BadRequestException;
+import greencity.exceptions.NotFoundException;
+import greencity.exceptions.http.AccessDeniedException;
+import greencity.exceptions.http.RemoteServerUnavailableException;
+import org.apache.commons.io.IOUtils;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class RetrieveMessageErrorDecoder implements ErrorDecoder {
+    @Override
+    public Exception decode(String methodKey, Response response) {
+        Map<String, Object> errorAttributes = new HashMap<>();
+        errorAttributes.put("message", "An error occurred");
+        errorAttributes.put("timestamp", "");
```

Comment:
why do you use an empty timestamp?

Reply:
please move `message` and `timestamp` to the constants

Reply:
edited